### PR TITLE
Implement auto import conflict resolution

### PR DESCRIPTION
**Pull request by bthesorceror**

### CODE REVIEW

File: src/visitors/semantic/utils.rs [hunk 1]
```diff
@@ -10,7 +10,10 @@ use flux::semantic::convert::convert_with;
 use flux::semantic::fresh::Fresher;
 use flux::semantic::nodes::Package;
 
-use libstd::analyze;
+use flux::analyze;
+
+// use core::analyze;
```
Comment (jsteenb2) [inline src/visitors/semantic/utils.rs:15]: can these comments be axed?

File: tests/handlers.rs [hunk 2]
```diff
@@ -392,7 +392,7 @@ speculate! {
 
                 assert_eq!(
                     signatures.len(),
-                    4,
```
Comment (jsteenb2) [inline tests/handlers.rs:395]: can you help me understand why this grew to `64`?

Comment (bthesorceror) [reply tests/handlers.rs:395]: the number of possible arguments increased for `from` so before we had 2 which results in 4 possible combinations now there are 6 arguments which is 6 possible combinations

File: src/visitors/semantic/utils.rs (hunk 1 above)
Comment (bthesorceror) [reply src/visitors/semantic/utils.rs:15]: yes!

**Pull request merged**